verification/curator-service/Dockerfile

```diff
@@ -31,9 +32,27 @@ COPY verification/curator-service/ui/. .
 ENV REACT_APP_LOGIN_URL /auth/google
 RUN npm run-script build
```

Comment:
Would be good to make these all `run` if we're in the business of changing them now.

Reply:
indeed, done